tests/h2_otel.t

```diff
@@ -47,6 +47,10 @@ http {
         batch_count 1;
     }
 
+	otel_resource_attr {
```

Comment:
Given, these attributes are not HTTP version related, I don't think we need to add them into h2/h3 tests.